### DISCUSSION TITLE
Failing to connect with Omnibridge and xDai Bridge

### DISCUSSION THREAD

**Discussion by april3636** [WalletConnect/walletconnect-monorepo#2977] [Developer Support]
Hello,

I am trying to connect my Safe Wallet with Omnibridge or xDaiBridge using Wallet Connect but nothing is happening. The Safe Wallet Help Desk said that there is an issue with Wallet Connect and I would like to know what the status is. Will this be fixed soon? Is there anything we can do in the meantime? I need to transfer some funds urgently to another wallet and I am stuck. Looking forward to your assistance with this.

**Comment by crypblizz8** [comment]
Hey there, this is being tracked [here](https://github.com/orgs/WalletConnect/discussions/2733) and we are currently testing the implementation before release. Closing this one so you can follow at the relevant discussion

**Comment by april3636** [comment]
Hello,

Thank you for your response. Apologies but I don’t understand the text. I
am very new to crypto and wallets etc. so in plain language could you tell
me what I can do to connect my Safe wallet with Omnibridge or Xdai bridge.
Is there a bug that you are currently working on? Is there any other option?

Many thanks in advance.

Best,
Ebru
